### PR TITLE
Add chain details page with list of members

### PR DESCRIPTION
**Pull request by navase**
First draft of chain details page. Added it under /chains/chainId, which might be useful for admins.

Check it out with for example: http://localhost:3000/chains/4dULvpW6kZO50nvNYOjl

Still to do:
- Authorization
- Sub menu with tab for member list and tab for chain details
- Editing and removing members
- CSV download

### CODE REVIEW

File: src/pages/chains/chain.tsx [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+type TParams = {
+  chainId: string;
+}
+
+interface IUsersState {
+  chain: IChain | null;
+  users: IUser[] | null;
+}
+
+class chainMemberList extends Component<RouteComponentProps<TParams>, IUsersState> {
```
Comment (timstokman) [inline src/pages/chains/chain.tsx:23]: ChainMemberList

File: src/App.tsx [hunk 2]
```diff
@@ -8,6 +8,7 @@ import themeFile from "./util/theme";
 import home from "./pages/home";
 import login from "./pages/login";
 import signup from "./pages/signup";
+import chain from "./pages/chains/chain";
```
Comment (timstokman) [inline src/App.tsx:11]: Gotta wonder why these components are not capitalized. This is against naming conventions.

File: src/util/firebase/chain.ts [hunk 3]
```diff
@@ -2,6 +2,18 @@ import db from "./firebaseConfig";
...
+  return await db.collection("chains")
+    .doc(chainId)
+    .get()
+    .then((doc) => {
+      return {
+        chain: doc.data() as IChain,
+        chainReference: doc.ref
+      };
+    });
+};
```
Comment (timstokman) [inline src/util/firebase/chain.ts:15]: You're mixing two different types of async processing here. The .get() call should already return a promise, you can you await that call. "then" uses the callback-style of async processing. You can mix them.. but it doesn't really make things clearer.

Some examples here: https://firebase.google.com/docs/firestore/query-data/get-data#node.js_6 -> click on javascript tabs.

Comment (navase) [reply src/util/firebase/chain.ts:11]: Thanks for the explanation and docs! I've tried to rewrite it, please take a look.

Comment (timstokman) [reply src/util/firebase/chain.ts:11]: Looks good now.

**Review by pim0223 — APPROVED** [review]

**Pull request merged**